app/src/main/java/org/simple/clinic/widgets/ViewFlipperWithDebugPreview.kt

```diff
@@ -3,20 +3,26 @@ package org.simple.clinic.widgets
 import android.content.Context
 import android.util.AttributeSet
 import android.widget.ViewFlipper
-
+import io.reactivex.Observable
+import io.reactivex.subjects.BehaviorSubject
+import io.reactivex.subjects.Subject
 import org.simple.clinic.R
 
 /** Exposes a way to set the displayed child in layout preview. */
 class ViewFlipperWithDebugPreview(context: Context, attrs: AttributeSet) : ViewFlipper(context, attrs) {
 
   private var childToDisplayPostInflate: Int = 0
 
+  private val displayedChildChangesSubject: Subject<Int> = BehaviorSubject.create()
```

Comment:
@rakshakhegde I updated this subject's type to `Behavior` so that the last value is always replayed on subscribe. This is in line with how RxBinding works.